st2common/st2common/runners/paramiko_ssh.py

```diff
@@ -698,8 +705,8 @@ def _connect(self, host, socket=None):
                  '_username': self.username, '_timeout': self.timeout}
         self.logger.debug('Connecting to server', extra=extra)
 
-        socket = socket or ssh_config_file_info.get('sock', None)
-        if socket:
+        self.socket = self.socket or ssh_config_file_info.get('sock', None)
```

Comment:
Since `self.socket` is only ever assigned `None`, this will always get pulled from `ssh_config_file_info.get('sock', None)`. Did you mean to mean this?

```python
self.socket = socket or ssh_config_file_info.get('sock', None)
            # ^^^^^^
            # not self.socket
```

Reply:
@eedgar

Reply:
This should be resolved.